packages/samples/mvp-dapp/contract/README.md

```diff
@@ -0,0 +1,62 @@
+# kuai mvp contract
+
+## Install dependencies
+
+- [ckb-cli](https://github.com/nervosnetwork/ckb-cli)
+- [capsule](https://github.com/nervosnetwork/capsule)
+
+## Configuration file modification
+
+By default, we have integrated several [networks](https://github.com/ckb-js/kuai/blob/develop/packages/core/src/constants.ts#L8) that can be used in the configuration file as follows.
+
+```js
+module.exports = {
+  ...
+  network: 'testnet' | 'mainnet',
+  jest: {
+      ...
+  },
+}
+
+```
+
+If we need to customize the network, we can modify the `kuai.config.js` file in the root of the project.
+
+```js
+module.exports = {
+  ...
+  network: 'testnet',
+  networks: {
+    testnet: {
+      rpcUrl: 'https://testnet.ckb.dev',
+      prefix: 'ckt',
+    }
+  },
+  jest: {
+      ...
+  },
+}
+
+```
```

Comment:
This step is not mandatory in my journey of deploying contract

Ref: https://github.com/ckb-js/kuai/issues/306